docs/reference_guides/model_libraries/generic/unit_models/stream_scaler.rst

```diff
@@ -0,0 +1,46 @@
+Stream Scaler Block
+===================
+
+Stream Scaler Blocks are used to adjust size of streams to represent, for example, a stream being split across several identical units, which are then all modeled as a single IDAES unit
+
+Degrees of Freedom
+------------------
+
+Stream Scaler blocks generally have zero degrees of freedom.
+
+Model Structure
+---------------
+
+Stream Scaler Blocks consists of a single StateBlock (named properties), each with an inlet and outlet port.
+
+Additional Constraints
+----------------------
+
+Stream Scaler Blocks write no additional constraints to the model.
```

Comment:
This is also not correct - the model is adding constraints.

Reply:
No, it actually isn't . It forwards `VarLikeExpressions` on the outlet port that have adjusted extensive variables. Additional `Constraints` arise from the additional arc constraints, but nothing internal to the unit model.

Reply:
Or rather, I should say no additional constraints* (besides those naturally occurring in `StateBlocks`).